# Generated by Django 2.0.8 on 2018-10-26 10:15

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('datasets', '0055_datasetrelease_taxonomy_node_stats'),
    ]

    operations = [
        migrations.CreateModel(
            name='ErrorReport',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('reviewed_at', models.DateTimeField(blank=True, null=True)),
                ('reviewed', models.CharField(choices=[('NR', 'Not Reviewed'), ('V', 'Valid'), ('NV', 'Not Valid')], default='NR', max_length=2)),
                ('annotation', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='datasets.GroundTruthAnnotation')),
                ('created_by', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.AlterUniqueTogether(
            name='errorreport',
            unique_together={('annotation', 'created_by')},
        ),
    ]
